incubator/mssql-linux/values.yaml

```diff
@@ -0,0 +1,29 @@
+ACCEPT_EULA:
```

Comment:
Values should be camel case and start with a lower case letter.

Reply:
Thanks.  Fixed